easybuild/easyconfigs/o/OpenMPI/OpenMPI-5.0.7_fix-sshmem-build-failure.patch

```diff
@@ -0,0 +1,28 @@
+Patch from https://github.com/open-mpi/ompi/issues/13103#issuecomment-2662258935
```

Comment:
@branfosj Can we update the patch description to point to https://github.com/open-mpi/ompi/pull/13105, which is the official fix?